web/src/components/pages/profile/info/info.tsx

```diff
@@ -160,7 +160,9 @@ function ProfilePage({
       locales: locales.filter(l => l.locale),
       visible: JSON.parse(visible.toString()),
       client_id: user.userId,
+      challenge_team: user.userClients[0].challenge_team,
```

Comment:
I think that we should save this onto `userFields` instead, so it's accessible like any other field. That probably involves threading it into the store at `/web/src/stores/user.ts`.

Reply:
maybe, but values in userFields come from the DB, now, challenge_team is like email, have not been persisted into DB.